scripts/build/bash/btfgen.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+TMPDIR=$1
+ARCH=$2
+FROM=$3
+OUTPUT=$4
+BPF_SO_PATTERN="^bpf\_[a-z0-9]+\.o"
+
+echo "btfhub-archive is a big archive project, maybe take some times..."
+git clone --depth 1 https://github.com/aquasecurity/btfhub $TMPDIR/btfhub
+git clone --depth 1 https://github.com/aquasecurity/btfhub-archive/ $TMPDIR/btfhub-archive/
+mv $TMPDIR/btfhub-archive/* $TMPDIR/btfhub/archive/
+
+each_all_bpf_so_file() {
+    result=""
+    for file in `ls -a $1`
+    do
+        if [ -d $1"/"$file ]
+        then
+            if [[ $file != '.' && $file != '..' ]]
+            then
+                each_all_bpf_so_file $1"/"$file
+            fi
+        elif [[ "$file" =~ $BPF_SO_PATTERN ]]
+        then
+          echo "-o $1/$file"
+        fi
+    done
+}
+
+${TMPDIR}/btfhub/tools/btfgen.sh -a ${ARCH} $(each_all_bpf_so_file $FROM)
+mkdir -p ${OUTPUT}
+cp -r ${TMPDIR}/btfhub/custom-archive/* ${OUTPUT}
```

Comment:
You actually copy some of the files from repo https://github.com/aquasecurity/btfhub-archive into our binary, please remember to add these in our binary license.

Reply:
The files under `custom-archive` are generated by `btfgen.sh`(btfhub tools), you mean we need to add https://github.com/aquasecurity/btfhub to the binary license? The `btfhub-archive` are only some archive binary files.  Am I right?

Reply:
> The files under `custom-archive` are generated by `btfgen.sh`(btfhub tools), you mean we need to add https://github.com/aquasecurity/btfhub to the binary license? The `btfhub-archive` are only some archive binary files. Am I right?

Correct me if I'm wrong, `btfgen.sh` "generates" `custom-archive` by selecting some of the files from https://github.com/aquasecurity/btfhub-archive , right? If so we should add license https://github.com/aquasecurity/btfhub-archive to binary license

Reply:
Yes, I take a look at this project. There are no licenses declared. So we ignore it temprory?

Reply:
If it doesn't have a license, we should not use it. Default, the owner has its 100% copyright, and doesn't grant to us and the foundation, even though everything is readable, doesn't mean open-source. 

It is better if you open an issue to ask the owner.

Reply:
OK, it is not urgent, let's hold this.

Reply:
Ok license file created in the btfhub-archive repository. Sorry for the delays. Let me know if there is anything else needed.

Reply:
Thanks!!!

Reply:
@kezhenxu94 @wu-sheng Hi, the license issue has been resolved, please recheck:)

Reply:
@rafaeldtinoco Thanks for fixing this so quickly, and licensed in APLv2.